pkgs/games/steam/steamcmd.nix

```diff
@@ -37,8 +37,9 @@ stdenv.mkDerivation {
   '';
 
   meta = with lib; {
-    description = "Steam command-line tools";
     homepage = "https://developer.valvesoftware.com/wiki/SteamCMD";
+    description = "Steam command-line tools";
```

Comment:
Why `description` has to be under `homepage`? (I mean why are you making this change)

Reply:
It doesn't matter, but that's the order they're in in most derivations

Reply:
Yeah, just leave it. I would have ignored it but I found another issue which we should fix.